app/services/rummager_link_sync.rb

```diff
@@ -0,0 +1,27 @@
+class RummagerLinkSync
```

Comment:
Although you will see some API's built just the way you have done it, nothing against it, just wanted to know what was the reason behind it.